src/Migrations/Version20190627203104.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190627203104 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'sqlite', 'Migration can only be executed safely on \'sqlite\'.');
+
+        $this->addSql('CREATE TABLE nomenclature_picture_set (nomenclature_id INTEGER NOT NULL, picture_set_id INTEGER NOT NULL, PRIMARY KEY(nomenclature_id, picture_set_id))');
+        $this->addSql('CREATE INDEX IDX_125B3C7390BFD4B8 ON nomenclature_picture_set (nomenclature_id)');
+        $this->addSql('CREATE INDEX IDX_125B3C73811FBB08 ON nomenclature_picture_set (picture_set_id)');
+        $this->addSql('CREATE TABLE picture_set (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, name VARCHAR(255) NOT NULL)');
+        $this->addSql('DROP INDEX IDX_799A3652C54C8C93');
```

Comment:
When dropping my db the script fail just here... There is no index named `IDX_799A3652C54C8C93`

Reply:
I don't understand why the migration has been currupted (same on my local version actually) maybe I made a mistake as I created several entities in the 2 previous branches. 
Anyway I recreate the migration and it should work as expected. Can you try again?